### ISSUE TITLE
Use multi threading

### ISSUE THREAD

**Issue by ViralBShah** [Circuitscape/Circuitscape.jl#197]
We should move to Julia multi threading and get ready for 1.3.

**Comment by ViralBShah** [comment]
Also, with Omniscape using multi-threading, it makes sense for Circuitscape to also be doing the same.

**Comment by ranjanan** [comment]
Blocking issue: I see this come up fairly often when I'm running it multi-threaded: https://github.com/JuliaMath/IterativeSolvers.jl/issues/272

**Comment by ViralBShah** [comment]
@ranjanan Can you post here what the current issue is, and how one should reproduce it?

**Comment by ViralBShah** [comment]
Never mind - I see it in #230.

**Comment by glaroc** [comment]
I'm really a Julia novice, but could Symbolics.jl be useful for implementing better parallelization and multi-threading? 
https://discourse.julialang.org/t/ann-symbolics-jl-a-modern-computer-algebra-system-for-a-modern-language/56251

**Comment by ViralBShah** [comment]
I think it is not related.

**Comment by glaroc** [comment]
You're right, I originally misunderstood the description of this package.
